homeassistant/components/weather/__init__.py

```diff
@@ -27,6 +27,10 @@
 ATTR_FORECAST_TEMP = 'temperature'
 ATTR_FORECAST_TEMP_LOW = 'templow'
 ATTR_FORECAST_TIME = 'datetime'
+ATTR_FORECAST_WIND_SPEED = 'wind_speed'
+ATTR_FORECAST_WIND_BEARING = 'wind_bearing'
+ATTR_FORECAST_DEW_POINT = 'dew_point'
+ATTR_FORECAST_PRESSURE = 'pressure'
```

Comment:
This component follow a internal standard. If you want extend this, you need first make a request for this in: https://github.com/home-assistant/architecture

After that you need extend the base class, demo and UI and update the documentation about this entity.

Reply:
Are you referring to dew point and pressure? I can remove those and discuss extending later. wind_speed and wind_bearing are already on the base class as properties and already in use by openweathermap as part of the forecast.

Reply:
I removed dew point and pressure. I noticed the ecobee is setting pressure as an attribute, so there might need to be a larger effort here, unless I'm missing the point (which I might be).

I'll submit something to the standard for those 2. I think having them available as both current and forecast properties is a value add for a weather component. Any other weather property you can think of that would be good to have available?

Reply:
The `weather` platforms should be on the "lowest common denominator" to make them exchangeable. It might be that one third-party service is delivering a value but none of the others. Thus it doesn't make much sense to include it.

For the full flexibility we have the weather `sensor`s which can be used if you want a specific value.

Feel free to start a discussion on https://github.com/home-assistant/architecture because what was the case yesterday could have changed today.